homeWork_1week.js

```diff
@@ -0,0 +1,234 @@
+    console.log(result)
+};
+
+deleteChars('Hello');
+deleteChars('A');
+
+const deleteChars2 = str => {
+    const result = str.split('').slice(1,str.length-1).join('');
+    console.log('результат: '+ result)
+};
+
+deleteChars2('Hello');
+deleteChars2('A');
+
+console.log('-----------------------')
+
+//Exersize 6 
+
+const getUpperCase = function(str){
+    const arrayStr = str.split('');
+    const newArray = [];
+    for(i=0; i<arrayStr.length; i++){
+        if(i===0) { 
+            newArray.push(arrayStr[i].toUpperCase()); 
+        }else newArray.push(arrayStr[i].toLowerCase());
+    }
+    const result = newArray.join('');
+    console.log(result);
+};
+
+getUpperCase('пиТеР');
+getUpperCase('javaScript');
+
+const getUpperCase2 = str => {
+    const arrayStr = str.split('');
+    const newArray = [];
+    for(i=0; i<arrayStr.length; i++){
+        if(i===0) { 
+            newArray.push(arrayStr[i].toUpperCase()); 
+        }else newArray.push(arrayStr[i].toLowerCase());
+    }
+    const result = newArray.join('');
+    console.log(`Правильное зачение: ${result}`);
+};
+
+getUpperCase2('пиТеР');
+getUpperCase2('javaScript');
+
+console.log('-----------------------')
+
+//Exersize 7
+
+const getInversetype = function(str){
+    const newArray = [];
+    for(i=0;i<str.length; i++){
+        if(typeof(str[i])==='number') {
+            newArray.push(String(str[i]))
+        } else if (typeof(str[i])==='string') {
+            newArray.push(Number(str[i]))
+        }
+    }
+    console.log(str)
+    console.log(newArray)
+};
+
+getInversetype([1,2,3,4,'6','7','8','9']);
+
+const getInversetype2 = str => {
+    const newArray = [];
+    for(i=0;i<str.length; i++){
+        if(typeof(str[i])==='number') {
+            newArray.push(String(str[i]))
+        } else if (typeof(str[i])==='string') {
+            newArray.push(Number(str[i]))
+        }
+    }
+    console.log(`Исходный массив:`)
+    console.log(str)
+    console.log(`Преобразованный массив:`)
+    console.log(newArray)
+}
+
+getInversetype2([1,2,3,4,'6','7','8','9']);
+
+console.log('-----------------------')
+
+//Exersize 8
+
+const getSumMassiv = function(str1,str2){
+    const newArray = str1.concat(str2);
+    console.log(newArray)
+};
+getSumMassiv([1,2,3],[4,5,6]);
+
+const getSumMassiv2 = (str1,str2) => {
+    const newArray = str1.concat(str2);
+    console.log(newArray)
+};
+
+getSumMassiv2([1,2,3],[4,5,6]);
+
+console.log('-----------------------')
+
+//Exersize 9
+
+const stringCutter = function(str, num){
+    const newArray = str.split('');
+    newArray.splice(num, str.length, '...')
+    const result = newArray.join('')
+    console.log(result)
+}
+
+stringCutter('Привет Мир!', 6);
+
+const stringCutter2 = (str, num) => {
+    const newArray = str.split('');
+    newArray.splice(num, str.length, '...')
+    const result = newArray.join('')
+    console.log(`первые ${num} выделенных букв из строки + многоточие: ${result}`)
```

Comment:
you can avoid parsing string to array and array to string back, if you will use `slice` string method